src/components/DatePicker/index.js

```diff
@@ -29,6 +29,8 @@ class Datepicker extends React.Component {
     componentDidMount() {
         // Adds nice native datepicker on web/desktop. Not possible to set this through props
         this.inputRef.setAttribute('type', 'date');
+        this.inputRef.setAttribute('min', this.props.minimumDate ? moment(this.props.maximumDate).format(CONST.DATE.MOMENT_FORMAT_STRING) : undefined);
+        this.inputRef.setAttribute('max', this.props.maximumDate ? moment(this.props.maximumDate).format(CONST.DATE.MOMENT_FORMAT_STRING) : undefined);
```

Comment:
Can we require these props instead of handling it with a ternary like this?

Reply:
Since this is an optional prop &` this.props.maximumDate` is an instance of Date. It doesn’t make sense to require these props.

Reply:
Also, a ternary check is there because only if the prop is available should format using `moment`. Because `moment(undefined )` will return current date. So I think it's better this way. 

Can I make it if statements?

```js
if (this.props.minimumDate){
     this.inputRef.setAttribute('min', moment(this.props.minimumDate).format(CONST.DATE.MOMENT_FORMAT_STRING));
}
if (this.props.maximumDate){
     this.inputRef.setAttribute('max', moment(this.props.maximumDate).format(CONST.DATE.MOMENT_FORMAT_STRING));
}
```

Also @Luke9389 , just now noticed there is a typo here, instead minimumDate ,  maximumDate is used in L32
<img width="1197" alt="Screenshot 2021-11-10 at 2 20 09 AM" src="https://user-images.githubusercontent.com/85645967/141002441-4786acde-44f9-432d-9c11-c14f8433946b.png">

I'll need to update this anyway. Let me if need to refactor the ternary to if statements

Reply:
> Since these are optional props. It doesn’t make sense to require these props.

Why shouldn't we require them?

Reply:
> Why shouldn't we require them?

Because `minimumDate` & `maximumDate` are optional props to the DatePicker component, which restricts the user to pick a date only between the min & max date available only. So it wouldn’t make sense to require those props. If made them required props we would be forced to give them min & max date in all implementation of the Date Picker component.

Reply:
We won't require both min & max date all the time, so let's leave it as it is.

Reply:
Corrected the typo.

@Luke9389 Let me know if you have any questions?

Reply:
Yea @Santhosh-Sellavel, I should have been more clear in my last comment. I was thinking, why wouldn't we want to require min & max dates for every date picker? But to keep things simple, I'll just drop that for now. 

Also, I do like the if statements better. 👍

Reply:
Updated the PR with ternary checks with if statements.